### PR TITLE
divide c11 atomic function by size to allow windows compatibility

### PR DESCRIPTION
**Pull request by sgaragagghu**

### CODE REVIEW

File: client/utils/atomic.h [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	switch(sizeof(uint_least8_t)){
+		case 1:
+			return _InterlockedExchange8((char volatile*)object, desired);
+		case 2:
+			return _InterlockedExchange16((short volatile*)object, desired);
+		...continue...
+}
+**********************************************************/
+
+#define atomic_init_uintptr(a,b) atomic_init(a,b)
```
Comment (jonano614) [inline client/utils/atomic.h]: it seems those definitions are the same

Comment (sgaragagghu) [reply client/utils/atomic.h]: changed it now =)

Comment (sgaragagghu) [reply client/utils/atomic.h]: @jonano614 merged with your implementation of stdatomic

Comment (jonano614) [reply client/utils/atomic.h]: Hi, great! I will test it and merge.

**Pull request merged**